ir_filters_multi_company/security/ir_filters.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
```

Comment:
```suggestion
<odoo>
```

Sorry for the noice : ignore that last comment. see @pedrobaeza remarks here : https://github.com/OCA/multi-company/pull/506#discussion_r1429125380

Reply:
OK, reverted